azure-batch/azure/batch/models/node_counts.py

```diff
@@ -55,6 +57,7 @@ class NodeCounts(Model):
         'running': {'required': True},
         'starting': {'required': True},
         'start_task_failed': {'required': True},
+        'leaving_pool': {'required': True},
```

Comment:
In order for this to be not breaking, this model has to be used only as output (i.e. from Azure to customer). Please confirm that no operation is taking this as input.

Reply:
Yes. This is response.